stereotype/model.py

```diff
@@ -148,6 +148,12 @@ def __getitem__(self, key):
             raise KeyError(key)  # Avoids getting methods via getitem
         return getattr(self, key)
 
+    def get(self, key, default=None):
+      try:
+        return self[key]
+      except KeyError:
+        return default
```

Comment:
```suggestion
        if key not in self.__slots__:
            return default
        value = getattr(self, key)
        return value if value is not Missing else default
```

Reasoning:
* It depends on the intended semantics of the `get` method (probably could use a comment), but if a required field was missing in the primitive data, returning default seems more logical to me than returning Missing.
* Since stereotype is a performance focused library, I'd rather avoid a try-except block + function call and just "inline" the `__getitem__` implementation.
* Strangely, the body of the function was indented by 6 spaces instead of 8

Reply:
I applied the suggested changes.